pkg/apis/scorecard/v1alpha3/test_types.go

```diff
@@ -40,6 +40,7 @@ type TestStatus struct {
 // Test specifies a single test run.
 type Test struct {
 	metav1.TypeMeta `json:",inline"`
+	Tstamp          string            `json:"tstamp,omitempty"`
```

Comment:
Is it possible to use [`metav1.Time`](https://github.com/kubernetes/apimachinery/blob/8303750571678b1e7ccef2b8b597b5286aab7eb7/pkg/apis/meta/v1/time.go#L24-L33) instead of a string?

Reply:
We can use a time object and convert to a string later when it's needed. I wanted to add this field because Xunit output's would like a timestamp on the test result.
1) Current plan is to time stamp upon the creation of the result
2) Seems reasonable to move it to the result struct instead